cognite_toolkit/common_function_code/README.md

```diff
@@ -0,0 +1,7 @@
+# Common function code
```

Comment:
Main motivation for putting this on the top level is importing it and running the function locally?

Reply:
Yes, and to ensure you have a single location for common code across all modules.